### ISSUE TITLE
`ignore` being undefined in `Deno.test` causes registration to fail

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno#18827]
`Deno.test({ name: "a", ignore: undefined, fn() {}})` works in Deno 1.32.4 and breaks in Deno 1.32.5

```
/m/a/P/g/d/x ❯❯❯ deno test test.ts
Check file:///mnt/artemis/Projects/github.com/denoland/x/test.ts
Uncaught error from ./test.ts FAILED

 ERRORS 

./test.ts (uncaught error)
error: TypeError: Error parsing args at position 0: missing field `ignore`
Deno.test({ name: "a", ignore: undefined, fn() {}})
     ^
    at Object.test (ext:cli/40_testing.js:675:30)
    at file:///mnt/artemis/Projects/github.com/denoland/x/test.ts:1:6
This error was not caught from a test and caused the test runner to fail on the referenced module.
It most likely originated from a dangling promise, event/timeout handler or top-level code.

 FAILURES 

./test.ts (uncaught error)

FAILED | 0 passed | 1 failed (11ms)

error: Test failed
```

I suspect #18563

**Comment by lucacasonato** [comment]
Already fixed by #18786. My bad for not checking.

**Comment by lucacasonato** [comment]
Duplicate of #18784
